packages/appium/lib/extension/index.js

```diff
@@ -85,29 +115,30 @@ export function getActivePlugins(pluginConfig, usePlugins = []) {
  * If the --drivers flag was given, this method only loads the given drivers.
  *
  * @param {import('./driver-config').DriverConfig} driverConfig - a driver extension config
+ * @param {number} maxParallelImports the maximum amount of plugins to import in parallel
  * @param {string[]} [useDrivers] - optional list of drivers to load
- * @returns {DriverNameMap}
+ * @returns {Promise<DriverNameMap>}
  */
-export function getActiveDrivers(driverConfig, useDrivers = []) {
-  return new Map(
-    _.compact(
-      Object.keys(driverConfig.installedExtensions)
-        .filter((driverName) => _.includes(useDrivers, driverName) || useDrivers.length === 0)
-        .map((driverName) => {
-          try {
-            log.info(`Attempting to load driver ${driverName}...`);
-            const DriverClass = driverConfig.require(driverName);
-            return [DriverClass, driverName];
-          } catch (err) {
-            log.error(
-              `Could not load driver '${driverName}', so it will not be available. Error ` +
-                `in loading the driver was: ${err.message}`
-            );
-            log.debug(err.stack);
-          }
-        })
-    )
-  );
+export async function getActiveDrivers(driverConfig, maxParallelImports, useDrivers = []) {
+  /** @type {string[]} */
+  let filteredDriverNames = [];
+  if (useDrivers.length === 0) {
```

Comment:
Might be super picky, but the equivalent check in `getActivePlugins` uses `if (_.isEmpty(usePlugins))`, so this could be changed to also use lodash.